tests/reference_emails/submitted-dummy.html

```diff
@@ -42,19 +42,20 @@
 disp = DispatcherAPI(url=&#34;PRODUCTS_URL/dispatch-data&#34;, instrument=&#34;mock&#34;)<br>
 <br>
 par_dict = {<br>
-    &#34;product_type&#34;: &#34;Real&#34;,<br>
+    &#34;DEC&#34;: -29.74516667,<br>
```

Comment:
I must say alphabetic order is less readable, many sequences are not in regular order (e.g. RA-DEC, product is important enough to be first, etc). But it does not matter so much. Of course it's important to have defined ordering. Maybe later we can add different sorting. Not now.

Reply:
I thought the same, I will add a dedicated TODO